app/controllers/api/v1x3/mixins/index_mixin.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V1x3
+    module Mixins
+      module IndexMixin
+        include Api::V1x2::Mixins::IndexMixin
```

Comment:
do we need a new show_mixin.rb for v1x3?

Reply:
Yes, otherwise we need to include `Api::V1x2::Mixins::IndexMixin` for all v1x3 controllers. It's kind of awkward.

Reply:
but I don't see it.

Reply:
Now we only need this: https://github.com/RedHatInsights/catalog-api/pull/859/files#diff-9866d1cd14a1a754b86a7d5c927d89d9e80e4d0e0bfed9accc637e52e91d7f01R4